scripts/storage-initializer/download.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
```

Comment:
Do we refer to other project, like kfserving, is  there a copyright issue?

Reply:
I have already add copyright of kubeflow.org and the original source code link